File: 2DerecedenPolinom.cpp
#include <iostream>
using namespace std;

int main() {
    
    
    double a = 0, b = 0, c = 0;
   
    cout << "A sayisini giriniz: ";
    cin >> a;
    cout << "B sayisini giriniz: ";
    cin >>b;
    cout << "C sayisini giriniz: ";
    cin >> c;
    
    double sonuc = (a*a) + (b*b) + (3 *c);
    
    cout << "Sonuc = " << sonuc;
    
    cin.get();
    cin.ignore();
    return 0;
}

File: 2.DerecedenPolinom.swift
//  Created by Kağan on 15.01.2017.
//  Copyright © 2017 Kağan Utku Kılıçlı. All rights reserved.

import Foundation


print("a sayisini giriniz:")
if var sayiA = Double(readLine(strippingNewline: true)!) {
    print("b sayisini giriniz:")
    if var sayiB = Double(readLine(strippingNewline: true)!) {
        print("c sayisini giriniz:")
        if var sayiC = Double(readLine(strippingNewline: true)!) {
            print("sonuç: \(pow(sayiA,2.0) + pow(sayiB,2.0) + (3*sayiC))")
        }
    }
}


